src/main/java/thePackmaster/powers/intothebreachpack/StormGenPower.java

```diff
@@ -0,0 +1,46 @@
+package thePackmaster.powers.intothebreachpack;
+
+import com.megacrit.cardcrawl.actions.defect.ChannelAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.orbs.Lightning;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import java.util.ArrayList;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.att;
+
+public class StormGenPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("StormGenPower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public StormGenPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+    }
+
+    @Override
+    public void atEndOfTurn(boolean isPlayer) {
+        ArrayList<AbstractMonster> withDebuff = new ArrayList<>();
+
+        for (AbstractMonster m : AbstractDungeon.getCurrRoom().monsters.monsters)
```

Comment:
This iterates through dead monsters as well.
```suggestion
        for (AbstractMonster m : Wiz.getEnemies())
```
This requires a new import, be sure to add that